comp/process/bundle.go

```diff
@@ -0,0 +1,38 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+// Package process implements the "process" bundle, providing components for the Process Agent
+//
+// The constituent components serve as utilities and are mostly independent of
+// one another.  Other components should depend on any components they need.
+//
+// This bundle does not depend on any other bundles.
+package process
+
+import (
+	"github.com/DataDog/datadog-agent/comp/process/containercheck"
+	"github.com/DataDog/datadog-agent/comp/process/processcheck"
+	"github.com/DataDog/datadog-agent/comp/process/runner"
+	"github.com/DataDog/datadog-agent/comp/process/submitter"
+	"github.com/DataDog/datadog-agent/pkg/util/fxutil"
+)
+
+// team: processes
+
+// Bundle defines the fx options for this bundle.
+var Bundle = fxutil.Bundle(
```

Comment:
Thank you, will keep for now since we'd be placing it in two subcommands - one for running the daemon, the other one for executing a check in a one off fashion.